recipes-bsp/u-boot/files/0010-gpio-da8xx_gpio-Fix-gpio-name-with-address.patch

```diff
@@ -0,0 +1,42 @@
+From 65a42d7cbf4e8b7248876df7d6f6afb8ee52b57c Mon Sep 17 00:00:00 2001
+From: chao zeng <chao.zeng@siemens.com>
+Date: Fri, 7 Jan 2022 10:03:56 +0800
+Subject: [PATCH 10/16] gpio: da8xx_gpio: Fix gpio name with address
+
+THe GPIO bank numbers do not appear in the device tree,
+so  make the gpio name based on the address
+(ie gpio@42110000_25 vs 25)
```

Comment:
Is it possible to create some id for the gpios, such as gpio0, gpio1?
Or is there any global gpio name space?

Reply:
the base address could be obtained from Device-tree
combined the base address with the number is a generic approach to named the
different gpio controller0